src/gui/screen_print_preview.cpp

```diff
@@ -313,13 +313,17 @@ static int screen_print_preview_event(screen_t *screen, window_t *window,
     }
 
     if (!suppress_draw && fs_did_filament_runout()) {
+        rect_ui16_t msg_box_size = gui_defaults.scr_body_no_foot_sz;
+        msg_box_size.y = 0;
+        msg_box_size.h += gui_defaults.header_sz.h;
         suppress_draw = true;
-        Sound_Play(eSOUND_TYPE_StandardAlert);
+
+        Sound_Play(eSOUND_TYPE_SingleBeep);
         const char *btns[3] = { N_("YES"), N_("NO"), N_("IGNORE") };
-        switch (gui_msgbox_ex(string_view_utf8::MakeNULLSTR(),
+        switch (gui_msgbox_ex(string_view_utf8::MakeRAM((const uint8_t *)gcode_file_name),
             _("Filament not detected. Load filament now? Select NO to cancel, or IGNORE to disable the filament sensor and continue."),
             MSGBOX_BTN_CUSTOM3,
-            gui_defaults.scr_body_no_foot_sz,
+            msg_box_size,
```

Comment:
what is this change?

Reply:
After discussion with @RadekVana new gui default rect is added - scr_fullscreen with position and dimensions for a full display screen, so creating new rect as was before is obsolete now. Btw. this was fix for BFW-1108

Reply:
ok